rpm/systemd.spec

```diff
@@ -258,8 +258,8 @@ CONFIGURE_OPTS=(
         -Dinstall-tests=false
 %endif
         # those also will not work during boostrap if enabled
-        -Dresolve=false
-        -Dmyhostname=false
```

Comment:
Please only enable resolved only in the full version of systemd. 

The mini variant should only include features which have to be present at built time for other packages to be built.

For reference please check the openSUSE spec file:
https://build.opensuse.org/projects/openSUSE:Factory/packages/systemd/files/systemd.spec?expand=1

Reply:
So within the first `%if` as `true` and in `%else` as `false`?

Reply:
Yeah looks good.